pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -432,3 +403,25 @@ func GenerateInviteWithKeyAndEndpoint(invite *Invitation) (string, error) {
 
 	return encodedExchangeInvitation(invite)
 }
+
+// AutoExecuteActionEvent is a utility function to execute events automatically. The function requires a channel to be
+// passed-in to listen for dispatcher.DIDCommEvent and triggers the callback. This is a blocking function and use
+// this function with a goroutine.
```

Comment:
Suggest including code snippet to show how auto execution is set up

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/4d12dff2d14ec87906c8c69146befde29694ebfe..3526a9756b25f098cf4801865b8a7d1f506eb69b#diff-f761c91e753eba1c42a68076df06ce12R412).